Unifying demo

pellecchialuigi opened the pull request:
as per @nopeppermint suggestions at #75
* Renamed Container files 
* Renamed run_demo_rpi.sh

Added a script file to backup the podman volume related to the sqlite3 database file.
This script is copied in the /etc/cron.daily folder of the machine running the api project by the run_demo.sh file and is activating an automated daily backup of the database.
The filename without extension is wanted. It is the format supported by cronjobs.

nopeppermint commented:
what's about the [Docker Volumes](https://basil-the-fusa-spice.readthedocs.io/en/latest/how_to_run_it.html#docker-volumes)
you want to include them as well in this script?

nopeppermint commented on run_demo.sh:
```diff
@@ -102,6 +115,11 @@ do
+echo -e "\n###################################################################"
+echo -e "### install cronjobs\n"
+
+cp -a misc/cronjobs/daily/. /etc/cron.daily/
```
if script is not executed as root, I think we need here a "sudo" before

at least on my pc I get an error:


```
cp: regular file '/etc/cron.daily/./basil-db-vol-backup' cannot be created: No authorization
cp: the times for '/etc/cron.daily/.' are retained: Operation not allowed
```

pellecchialuigi replied:
make sense...
what if we add in the documentation to use sudo running the run_demo.sh script?
Is that PR addressing your backup request?

nopeppermint commented:
what do you think about the pip warning:

```
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
```

nopeppermint commented:
otherwise the script seems to work, you might want to add tests for all options..

pellecchialuigi commented:
For the volume mount issue, calling the unmount just before should work also if it is not yet mounted.

pellecchialuigi commented:
I added color to the run_demo.sh scripts executions to better distinguish each sections.

pellecchialuigi commented:
for the volume mount issue, instead of unmounting and remounting them I just added **or true**
I also added an alert message to the user if the basil-db-vol already exists because in that case we cannot apply the admin password selected from the user (or the default configuration) as the db comes from the existing volume.
